### DISCUSSION TITLE
Manjaro xfce problem

### DISCUSSION THREAD

**Discussion by Saleh-Mohey** [termux/termux-packages#9270] [Q&A]
How can i install Manjaro XFCE?
![Screenshot_٢٠٢٢-٠٣-٠٣-٠٠-٢٧-٠٥-٩٦٢_com termux](https://user-images.githubusercontent.com/53252533/156464662-8ea046de-027f-42a3-8b4f-592c33b00cf6.jpg)

**Comment by ghost** [comment]
`pkg install pv proot wget` and try again

**Reply by Saleh-Mohey** [reply]
Thanks bro
